src/cachers/collections/organisations.js

```diff
@@ -1,18 +1,49 @@
 import baseData from './index.js';
 import organizationsMixin from '../../mixins/europeana/entities/organizations.js';
+import { langMapValueForLocale } from '../../plugins/europeana/utils.js';
+import { createEuropeanaApiClient } from '../utils.js';
 
-const PICK = ['slug', 'prefLabel'];
+const PICK = ['slug', 'recordCount', 'nativeLabel', 'nonNativeEnglishLabel'];
 const INTERNATIONALISE = (entities) => entities.map((entity) => ({
   ...entity,
-  prefLabel: organizationsMixin.methods.organizationEntityNativeName(entity)
```

Comment:
I'd be more inclined to (once again) cache the full prefLabel here, and have the front-end responsible for picking out what it wants with the mixin methods.

That may in fact make the whole `INTERNATIONALISE` method redundant here.

Reply:
I was doubting where to handle this too. I have refactored it to handle the different language labels in the EntityTable component.